ethcore/src/snapshot/block.rs

```diff
@@ -23,7 +23,7 @@ use hash::keccak;
 use views::BlockView;
 use rlp::{DecoderError, RlpStream, UntrustedRlp};
 use util::{Bytes, H256};
-use util::triehash::ordered_trie_root;
+use triehash::ordered_trie_root;
```

Comment:
used only once in the code, could be referenced by global path instead of `use`, as in the test

Reply:
Is there some rules for the usage of `use` in parity project? (it currently seems not though)
For example:
* make the self-host libs a group and third party libs a group
* sort it alphabetically within the group
* make it clear when to use `*`
* make it clear when to use global path
* others

I think we can make those rules in the CONTRIBUTION.md and make contributors to follow those rules.

Reply:
We don't have such rules currently, although I totally agree that we should have them